src/main/java/com/amazon/dlic/auth/http/saml/AuthTokenProcessorHandler.java

```diff
@@ -411,7 +411,7 @@ private String extractSubject(SamlResponse samlResponse) throws Exception {
         if (samlRolesSeparator != null) {
             values = splitRoles(values);
         } else {
-            values = trimRoles(values);
+            //values = trimRoles(values);
```

Comment:
Why don't we just remove the entire IF statement?

Reply:
we should remove the else statement alltogether.